mw-webapp/src/logic/waysTable/FavoriteWaysTable.tsx

```diff
@@ -0,0 +1,42 @@
+import {useEffect, useState} from "react";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
+import {columns} from "src/logic/waysTable/columns";
+import {WaysTable} from "src/logic/waysTable/WaysTable";
+import {WayPreview} from "src/model/businessModelPreview/WayPreview";
+
+/**
+ * Favorite ways table props
+ */
+interface FavoriteWaysTableProps {
+
+  /**
+   * User Uuid
+   */
+  uuid: string;
+}
```

Comment:
Each table has the same interface with only one prop uuid/ What do you think about create one common props for all waysTables? Is it good or bad idea?